README.md

```diff
@@ -173,19 +178,264 @@ The explanation functions accept both models and pipelines as input. If a model
+                               LGBMExplainableModel, 
+                               augment_data=True, 
+                               max_num_of_augmentations=10, 
+                               features=breast_cancer_data.feature_names, 
+                               classes=classes)
+    ```
+   or
+
+    ```python
+    from interpret.ext.blackbox import PFIExplainer 
+    
+    # "features" and "classes" fields are optional
+    explainer = PFIExplainer(model, 
+                             features=breast_cancer_data.feature_names, 
+                             classes=classes)
+    ```
+## Overal (Global) feature importance values
+
+Get the global feature importance values.
+    
+```python
+
+# you can use the training data or the test data here
+global_explanation = explainer.explain_global(x_train)
+
+# if you used the PFIExplainer in the previous step, use the next line of code instead
+# global_explanation = explainer.explain_global(x_train, true_labels=y_test)
+
+# sorted feature importance values and feature names
+sorted_global_importance_values = global_explanation.get_ranked_global_values()
+sorted_global_importance_names = global_explanation.get_ranked_global_names()
+dict(zip(sorted_global_importance_names, sorted_global_importance_values))
+
+# alternatively, you can print out a dictionary that holds the top K feature names and values
+global_explanation.get_feature_importance_dict()
+```
+
+## Instance-level (local) feature importance values
+Get the local feature importance values: use the following function calls to explain an individual instance or a group of instances. Please note that PFIExplainer does not support local explanations.
+
+```python
+# explain the first data point in the test set
+local_explanation = explainer.explain_local(x_test[0])
+
+# sorted feature importance values and feature names
+sorted_local_importance_names = local_explanation.get_ranked_local_names()
+sorted_local_importance_values = local_explanation.get_ranked_local_values()
+```
+
+or
+
+```python
+# explain the first five data points in the test set
+local_explanation = explainer.explain_local(x_test[0:5])
+
+# sorted feature importance values and feature names
+sorted_local_importance_names = local_explanation.get_ranked_local_names()
+sorted_local_importance_values = local_explanation.get_ranked_local_values()
+```
+
+
+## Raw feature transformations
+
+Optionally, you can pass your feature transformation pipeline to the explainer to receive explanations in terms of the raw features before the transformation (rather than engineered features). If you skip this, the explainer provides explanations in terms of engineered features.
+
+The format of supported transformations is same as the one described in [sklearn-pandas](https://github.com/scikit-learn-contrib/sklearn-pandas). In general, any transformations are supported as long as they operate on a single column and are therefore clearly one to many. 
```

Comment:
I think we support many-to-many mapping too by turn on one setting. Forget about the parameter name. @hchandola

Reply:
we do. Its called `allow_all_transformations`